### PR TITLE
add content for new WPF APIs

### PR DESCRIPTION
**Pull request by mairaw**
Related to https://github.com/dotnet/docs/issues/4876

### CODE REVIEW

File: xml/Microsoft.Build.Tasks.Windows/GenerateTemporaryTargetAssembly.xml [hunk 1]
```diff
@@ -199,9 +199,19 @@
...
+        <value>
+          <see langword="true" /> if debugging information is enabled for the <see cref="T:Microsoft.Build.Tasks.Windows.GenerateTemporaryTargetAssembly" /> Task; otherwise, <see langword="false" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:Microsoft.Build.Tasks.Windows.GenerateTemporaryTargetAssembly.GenerateTemporaryTargetAssemblyDebuggingInformation%2A> property is an optional task parameter. It's a diagnostic parameter, and it defaults to `false`.  
+  
+ At this time, the only debugging information that is generated consists of the temporary project that is created to generate the temporary target assembly. The temporary project is normally deleted at the end of the MSBUILD task. When <xref:Microsoft.Build.Tasks.Windows.GenerateTemporaryTargetAssembly.GenerateTemporaryTargetAssemblyDebuggingInformation%2A> is enabled, the temporary project is retained for inspection by the developer.
```
Comment (rpetrusha) [inline xml/Microsoft.Build.Tasks.Windows/GenerateTemporaryTargetAssembly.xml]: I'm not sure that "At this time" is helpful; I'd delete it.
MSBUILD --> MSBuild

File: xml/System.Windows.Diagnostics/ResourceDictionaryDiagnostics.xml [hunk 2]
```diff
@@ -131,9 +131,9 @@
         <Parameter Name="dictionary" Type="System.Windows.ResourceDictionary" />
       </Parameters>
       <Docs>
-        <param name="dictionary">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="dictionary">A resource dictionary.</param>
+        <summary>Gets the framework element owners of a specified resource dictionary.</summary>
+        <returns>The framework element owners of <paramref name="dictionary" />. If there are no freamework element owners, the method returns <see langword="null" />.</returns>
```
Comment (rpetrusha) [inline xml/System.Windows.Diagnostics/ResourceDictionaryDiagnostics.xml]: freamework --> framework

File: xml/System.Windows.Diagnostics/ResourceDictionaryDiagnostics.xml [hunk 3]
```diff
@@ -154,9 +154,9 @@
         <Parameter Name="uri" Type="System.Uri" />
       </Parameters>
       <Docs>
-        <param name="uri">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="uri">The source URI. The method return <see langword="null" /> if no resource dictionaries were created from <c>uri</c></param>
```
Comment (rpetrusha) [inline xml/System.Windows.Diagnostics/ResourceDictionaryDiagnostics.xml]: I think that "The method return..." should be deleted. The full(er) set of conditions is given in the return value section.

**Comment by mairaw** [comment]
I've addressed all your feedback @rpetrusha. Thanks!

**Pull request merged**